tests/test_utils.py

```diff
@@ -160,6 +160,33 @@ def test_negative_verify_jwt(self):
                 with self.assertRaises(DSSException):
                     security.verify_jwt(jwt)
 
+    def test_custom_email_claims(self):
+        self.addCleanup(self.restore_email_claims, os.environ.pop('OIDC_EMAIL_CLAIM', 'EMPTY'))
+        email = 'email@email.com'
+        email_claim = 'email@claim.com'
+        tests = [
+            ({'email': email, Config.get_OIDC_email_claim(): email_claim}, email_claim),
+            ({Config.get_OIDC_email_claim(): 'email@claim.com'}, email_claim),
+            ({'email': email}, email)
+        ]
+
+        for param, result in tests:
+            with self.subTest(f"no custom claim {param}"):
+                self.assertEqual(security.get_token_email(param), result)
+
+        os.environ['OIDC_EMAIL_CLAIM'] = 'TEST_CLAIM'
+        for param, result in tests:
+            with self.subTest(f"custom claim {param}"):
+                self.assertEqual(security.get_token_email(param), result)
+
+        with self.subTest("missing claim"):
+            with self.assertRaises(DSSException):
```

Comment:
Can you test more specifically here that it's a 401 exception?

Reply:
Will do